doc/conf.py

```diff
@@ -36,7 +36,7 @@
 from tribler_common.dependencies import Scope, get_dependencies
 from tribler_common.patch_import import patch_import
 
-modules_to_mock = set(get_dependencies(scope=Scope.core)) | {'libtorrent', 'validate'}
+modules_to_mock = set(get_dependencies(scope=Scope.core)) | {'libtorrent', 'validate', 'faker', 'sentry_sdk'}
```

Comment:
nit: this is also should works:

```suggestion
modules_to_mock = set(get_dependencies(scope=Scope.core)) | set(get_dependencies(scope=Scope.common)) | {'libtorrent', 'validate'}
```